apps/examples/src/styles.css

```diff
@@ -88,12 +86,65 @@ html,
 	}
 }
 
+.accordion__trigger{
+	border: none;
+	background: none;
+	cursor: pointer;
+	width: 100%;
+	text-align: left;
+	font-size: 10px;
+}
+
+.accordion__trigger__chevron{
+	color:rgba(0, 0, 0, 0.3);
+	transition: transform 300ms;
+	-webkit-transition: transform 300ms;
+	-moz-transition: transform 300ms;
+	-ms-transition: transform 300ms;
+	-o-transition: transform 300ms;
+}
+
+.accordion__trigger[data-state='closed'] > div > .accordion__trigger__chevron{
+	transform: rotate(-90deg);
+	-webkit-transform: rotate(-90deg);
+	-moz-transform: rotate(-90deg);
+	-ms-transform: rotate(-90deg);
+	-o-transform: rotate(-90deg);
+}
+
+
+.accordion__trigger__heading{
+	font-size: 14px;
+	font-weight: 500;
+	color:rgba(0, 0, 0, 0.5);
+	margin-top: 0.25rem;
+	margin-bottom: 0.25rem;
+}
+
+.accordion__content{
+	display: flex;
+	flex-direction: row;
+}
+.accordion__content__separator{
+	border-right: 1px solid #e8e8e8;
```

Comment:
would love to see these colors in a consolidated variable somewhere but i won't press you on this if it's not really strictly done in this app